### PR TITLE
Add original conference paper link to docs

### PR DESCRIPTION
**Pull request by ENUMERA8OR**
Added the link to the original conference paper link to index.md file.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -18,6 +18,8 @@
 
 [Mesa] is an Apache2 licensed agent-based modeling (or ABM) framework in Python.
 
+For the original conference paper click here: http://conference.scipy.org.s3-website-us-east-1.amazonaws.com/proceedings/scipy2015/jacqueline_kazil.html
```
Comment (EwoutH) [inline docs/index.md:21]: Currently is doesn't render nicely, what do you think of this?
```suggestion
The original conference paper is [available here](http://conference.scipy.org.s3-website-us-east-1.amazonaws.com/proceedings/scipy2015/jacqueline_kazil.html).
```

**Review by EwoutH — CHANGES_REQUESTED** [review]
Hi Aditya!

Thanks for your PR, and sorry to get back a bit late. This is a useful addition. I have one small suggestion to make it render beautiful on Read the Docs. You can preview it here: https://mesa--2160.org.readthedocs.build/en/2160/

File: docs/index.md (hunk 1 above)
Comment (ENUMERA8OR) [reply docs/index.md:21]: Okay will do that anything else that catches your mind please share thanks.

**Comment by ENUMERA8OR** [comment]
I have updated it please check @EwoutH .

**Review by EwoutH — APPROVED** [review]
Looks good, thanks!

**Pull request merged**